ui/src/layouts/BuildPublication.tsx

```diff
@@ -304,7 +291,7 @@ const BuildPublication: React.FC<BuildPublicationProps> = (props) => {
         [checkRequiredApproval, isReadyToPreview, store]
     );
 
-    const coAuthorsToApprove = useMemo(() => CheckCoAuthorsToApprove(store).ready, [CheckCoAuthorsToApprove, store]);
+    const hasCoAuthors = store.coAuthors.length > 1 ? true : false;
```

Comment:
why not simply "const hasCoAuthors = store.coAuthors.length > 1" ?

Reply:
why indeed 👍